modules/secure-cloud-function-core/main.tf

```diff
@@ -77,6 +105,7 @@ module "cloud_function" {
   storage_source      = var.storage_source
   service_config      = var.service_config
   docker_repository   = google_artifact_registry_repository.cloudfunction_repo.id
+  # worker_pool         = google_cloudbuild_worker_pool.pool.id
```

Comment:
nit: commented code

Reply:
This will require to be uncommented when the secure web proxy pr is ready.